docs/tempo/website/operations/architecture.md

```diff
@@ -10,12 +10,13 @@ A collection of documents that detail Tempo architectural decisions and operatio
 
 # Architecture
```

Comment:
^ I don't think these weights are working like we want them to, within a "main section" the titles seem to just be sorted in alphabetical order

Reply:
They do work (just tested) but nothing else in that subsection has a weight so they're all alphabetical. If we do want to do custom ordering in the drop down menus we can.

Reply:
ok, yeah custom ordering would be nice, same as the order we see on the index page for this section